9jaswag/PostIt
# #148287761 Maintain defined standards

#### What does this PR do?

- fix ESLint errors so as to maintain set standards
- remove anchor tags and replace them with React ```Link```
- reconfigures Webpack to bundle CSS better
- remove all ```console.log``` in the app
- add test cases to increase coverage
- Implement other feedbacks
- Fix reloading bug on ```PostMessage component```
- Implement ```remove user from group``` feature

#### Description of Task to be completed?
- Maintaining all standards set in tests, API documentation etc., implement the other features (server-side and client-side) in the roadmap defined in Pivotal Tracker.

#### How should this be manually tested?
- Visit the app to see that it no longer reloads
- View the app page source to see how Webpack handles CSS
- Check browser's console to see nothing is logged to the console
- Check coveralls.io to see the increase in code coverage
- Check CodeClimate to see increase in code climate

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/148287761
